dev/kube/ig-tools/yaml-memory.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
```

Comment:
Does this need to be a separate file? Why not include it in max-memory? Is max-memory output useful without this yaml converter?

Reply:
The separation allows me to
  - feed something manually written between max detection and conversion steps.
  - And another transformation too, must importantly a transform taking the max data and padding it.

Reply:
Ok - maybe call it something else then? like `max-memory-as-yaml.sh` ?
I think `yaml-memory` is not very descriptive, plus there's also `max-memory`, which makes it more confusing.

Reply:
Will work on a better name. Some of the confusion will be gone with the other scripts moving to `kubecf-tools`.

Reply:
See f96ab53a9b3968104baa30518261ab915377866b for new name (`make-memory-config.sh`).